addons/skin.estuary/xml/Home.xml

```diff
@@ -969,8 +907,12 @@
 					<width>480</width>
 					<height>110</height>
 					<top>100</top>
-					<onup>noop</onup>
-					<ondown>9000</ondown>
+					<onup>SetFocus(9000)</onup>
+					<onup>PageDown</onup>
+					<onup>PageDown</onup>
+					<ondown>SetFocus(9000)</ondown>
+					<ondown>PageUp</ondown>
+					<ondown>PageUp</ondown>
```

Comment:
guess it would be nice if we could do something like SetFocus(9000,0,absolute) to focus the actual first item, instead of the first visible item.

perhaps something for v18: https://github.com/ronie/xbmc/commit/d9e7a35825e1417f879ad3e274b3ed426e8f3256